src/types/OphanTypes.ts

```diff
@@ -1,9 +1,18 @@
+import * as z from 'zod';
+
 export type OphanProduct =
     | 'CONTRIBUTION'
     | 'MEMBERSHIP_SUPPORTER'
     | 'DIGITAL_SUBSCRIPTION'
     | 'PRINT_SUBSCRIPTION';
 
+export const ophanProductSchema = z.enum([
```

Comment:
It's not beautiful, but you can avoid duplication with:

```
const ophanProductValues = [
    'CONTRIBUTION',
    'MEMBERSHIP_SUPPORTER',
    'DIGITAL_SUBSCRIPTION',
    'PRINT_SUBSCRIPTION',
];
export type OphanProduct = typeof OphanProductValues[number];
export const ophanProductSchema = z.enum(ophanProductValues);
```

Reply:
I guess there's a typescripty reason for it, but you actually can't with `zod` https://github.com/colinhacks/zod#zod-enums

Reply:
ah that's a shame!

Reply:
Although, in these instances we could use zod to infer the type
```ts
export type OphanProduct = z.infer<typeof ophanProductSchema>();
```

Reply:
For enums I think that makes sense, it's preferable to duplicating these lists of values. I'm assuming our IDEs will be happy with the inferred type...

Reply:
Yeah looks like it infers the type exactly how you want it to: 

<img width="881" alt="Screenshot 2021-05-19 at 10 04 37" src="https://user-images.githubusercontent.com/17720442/118786500-c6300c00-b889-11eb-92d1-724e86c51cf3.png">